pipeline/notifications/new_message_render.py

```diff
@@ -0,0 +1,41 @@
+from typing import Set
+
+from pipeline.notifications.render_notification import (
+    InvalidNotificationTypeException,
+    RenderNotification,
+)
+from pipeline.types import NotificationInfo, NotificationType
+
+
+class NewMessageRender(RenderNotification):
+    def __init__(self):
+        pass
+
+    def format_messager_names(self, inp: Set[str]) -> str:
```

Comment:
Since this method does not use any state in the class (it never accesses `self`) and it is not part of the external functions of this class, it does not have to be an instance method.

So, you could move it to be its own function in this file file and remove `self`.

There is also a decorator called `@classmethod` that marks class methods that don't rely on any state. But that would only make sense if this were an external function that we expect users of this class to have access to. In this case, it is a function used only internally, so we would take it out of the class entirely.